README.md

```diff
@@ -15,6 +15,13 @@ npm run dev-production
 
 Now view localhost:4444 in your browser.
 
+**Obs:** If you are using another port for your estuary node other than the default port (`localhost:3004`), go to `common/constants.ts` and change the following line to return the address in which you have your server running
+```bash
+return 'http://localhost:3004';
```

Comment:
FWIW, I also added a `ESTUARY_API` env var that would make this a bit easier instead of needing to change code :)

```suggestion
**Note:** If you are using another port for your estuary node other than the default port (`localhost:3004`), you can set the `ESTUARY_API` environment variable to point to the right URL. for example. to change to port 9999
```bash
$ ESTUARY_API=http://localhost:9999 npm run dev